src/java.base/share/classes/java/lang/System.java

```diff
@@ -124,10 +124,19 @@ private System() {
      *
      * @apiNote
      * The typical approach to read character data is to wrap {@code System.in}
-     * within an {@link java.io.InputStreamReader InputStreamReader} or other object
-     * that handles character encoding. After this is done, subsequent reading should
-     * use only the wrapper object; operating directly on {@code System.in} results
-     * in unspecified behavior.
+     * within the object that handles character encoding. After this is done,
+     * subsequent reading should use only the wrapper object; operating directly
+     * on {@code System.in} results in unspecified behavior.
```

Comment:
Sorry, I think I suggested this wording initially, but I now think it needs to be changed. I'd suggest changing the text after the semicolon to

> continuing to operate directly on {@code System.in} results in unspecified behavior.

Reply:
Done.